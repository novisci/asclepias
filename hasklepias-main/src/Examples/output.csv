tagSet,facts,begin,end
"home",WasAfter,0,4
"not home",WasAfter,1,10
"home",WasAfter,0,20
