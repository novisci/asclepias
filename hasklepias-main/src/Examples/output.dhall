List { tagSet : Text, facts : < WasBefore | WasAfter >, begin : Integer, end : Integer}
