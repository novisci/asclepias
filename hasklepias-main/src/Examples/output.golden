"[[[0,4],{\"getFacts\":\"WasAfter\",\"getSource\":null,\"getTagSet\":[\"home\"]}],[[1,10],{\"getFacts\":\"WasAfter\",\"getSource\":null,\"getTagSet\":[\"not home\"]}],[[0,20],{\"getFacts\":\"WasAfter\",\"getSource\":null,\"getTagSet\":[\"home\"]}]]"